src/join/composer.jl

```diff
@@ -559,6 +653,10 @@ change in future releases.
   in `on` columns; if equal to `:equal` then `missing` is allowed and missings are
   matched; if equal to `:notequal` then missings are dropped in `df1` and `df2`
   `on` columns; `isequal` is used for comparisons of rows for equality
+- `sort` : if `nothing` (the default) the order of rows in the result is
+   undefined and may change in the future releases. If `:left` then the order of
```

Comment:
> may change in the future releases 

Should be "in the future" or "in future releases"

Reply:
What would be the difference in meaning? I would think it is the same. If some change happens it does so in a release.

Reply:
I am not a native speaker - so I might be wrong.

Reply:
Yes - the intention is that we do not plan to make such changes, but we do want to be allowed to make them in the future if we need to. I will adjust the wording to `"in future releases"` then. Right?

Reply:
I don't know what a linguist would say but my first-pass gut feeling is

- "in the future releases" might be used when referring to future releases that have been introduced already in the preceding context, perhaps to narrow the scope of what you're talking about. For example, "We have planned releases 1.3-1.9 and released 1.3-1.4. In the future releases, we will continue to add features."
- "in future releases" does not assume that there will be future releases. Any changes will be made in a future release, but the set of future releases may be empty. I think this is closest to your intended meaning.
- "in the future" does not specify that any changes would be made in a release.

Reply:
> the intention is that we do not plan to make such changes, but we do want to be allowed to make them in the future if we need to. I will adjust the wording to "in future releases" then. Right?

Yes